shared/uavobjectdefinition/hwshared.xml

```diff
@@ -67,7 +67,7 @@
 		</field>
 		<field name="MagOrientation" units="function" type="enum" elements="1" options="Top0degCW,Top90degCW,Top180degCW,Top270degCW,Bottom0degCW,Bottom90degCW,Bottom180degCW,Bottom270degCW" defaultvalue="Top0degCW"/>
 		<field name="USB_HIDPort" units="function" type="enum" elements="1" options="USBTelemetry,Disabled" defaultvalue="USBTelemetry"/>
-		<field name="USB_VCPPort" units="function" type="enum" elements="1" options="USBTelemetry,ComBridge,DebugConsole,Disabled" defaultvalue="Disabled"/>
+		<field name="USB_VCPPort" units="function" type="enum" elements="1" options="USBTelemetry,ComBridge,DebugConsole,MSP,Disabled" defaultvalue="MSP"/>
```

Comment:
Yes.  It's a definite RAM cost.

Advantage would be that passthrough would work out of box, and VCP telemetry would work out of box, too (could be a step along the way to transitioning away from HID telemetry).

In any case, it's not functional because every target currently overrides.